Autoscend breaks when trying to do seal clubbing rituals overdrunk

# Context

* KolMafia version: r27893
* Class: SC
* Run type (Casual/Normal/Hardcore/Aftercore): Normal
* Challenge Path (Standard/Path Name/None): Shrunken
* Number of Ascensions: 210

# Expected/Desired Behavior

When running autoscend overdrunk at the end of the day, script does not exit early

# Current Behavior

```
[WARNING] I am overdrunk
[INFO] Starting bedtime: Pulls Left: 0
[DEBUG] Could not use free combats because you have none
[DEBUG] lookupFamiliarDatafile is checking for type [stat]
Verifying ingredients for figurine of an ancient seal (1)...
Purchasing figurine of an ancient seal (1 @ 1 worthless item)...
Visiting Hermit...
Visiting the Hermit...
You acquire an item: figurine of an ancient seal
Preference _concoctionDatabaseRefreshes changed from 1211 to 1212
Hermit successfully looted!
Successfully created figurine of an ancient seal (1)
Purchasing seal-blubber candle (2 @ 90 = 180)...
You spent 180 Meat
You acquire seal-blubber candle (2)
Preference _concoctionDatabaseRefreshes changed from 1212 to 1213
Purchases complete.
Wielding oversized pipe...
Preference _concoctionDatabaseRefreshes changed from 1213 to 1214
Equipment changed.
[DEBUG] Removing "-equip oversized pipe" from current maximizer statement
[DEBUG] Adding "-weapon" to current maximizer statement
nextAdventure => Noob Cave
Preference nextAdventure changed from The Fungus Plains to Noob Cave
[INFO] Starting preadventure script...
[DEBUG] Adventuring at Noob Cave
[INFO] Equipping mafia thumb ring to slot acc3
Preference _auto_maximize_equip_acc3 changed from to mafia thumb ring
[DEBUG] Adding "-10ml" to current maximizer statement
[DEBUG] Removing "-equip mafia thumb ring" from current maximizer statement
[DEBUG] Adding "+equip mafia thumb ring" to current maximizer statement
[DEBUG] Adding "+50bonus spring shoes" to current maximizer statement
[DEBUG] Adding "+200bonus mafia thumb ring" to current maximizer statement
Maximizer: 5item,meat,0.5initiative,0.1da 1000max,dr,0.5all res,1.5mainstat,-fumble,mox,0.4hp,0.2mp 1000max,3mp regen,1.5weapon damage,0.75weapon damage percent,1.5elemental damage,2familiar weight,5familiar exp,-equip deceased crimbo tree,-equip makeshift garbage shirt,-weapon,-10ml,+equip mafia thumb ring,+50bonus spring shoes,+200bonus mafia thumb ring
Maximizing...
24 combinations checked, best score 4,977.67
Checkpoints cleared.
[DEBUG] Going into High or Standard ML Zone with ML: 0
Trying to adv in [Noob Cave] while overdrunk... Stop it.
Stack trace:
  at auto_pre_adventure (auto_pre_adv.ash:982)
  at main (auto_pre_adv.ash:1016)
[ERROR] Error running auto_pre_adv.ash, setting auto_interrupt=true
```

--- a/RELEASE/scripts/autoscend.ash
+++ b/RELEASE/scripts/autoscend.ash
@@ -1154,6 +1154,11 @@ boolean Lsc_flyerSeals()
 	{
 		return false;
 	}
+	// although seals can be fought drunk, it complicates code without serving a purpose
+	if(my_inebriety() > inebriety_limit())
+	{
+		return false;
+	}
 	if (internalQuestStatus("questL12War") != 1)
 	{
 		return false;

--- a/RELEASE/scripts/autoscend/auto_bedtime.ash
+++ b/RELEASE/scripts/autoscend/auto_bedtime.ash
@@ -619,7 +619,8 @@ boolean doBedtime()
 
 	while(LX_freeCombats());
 
-	if((my_class() == $class[Seal Clubber]) && guild_store_available())
+	// although seals can be fought drunk, it complicates code without a meaningful benefit
+	if((my_class() == $class[Seal Clubber]) && guild_store_available() && my_inebriety() <= inebriety_limit())
 	{
 		handleFamiliar("stat");
 		int oldSeals = get_property("_sealsSummoned").to_int();

